scripts/build-issues.js

```diff
@@ -0,0 +1,149 @@
+const { writeFileSync } = require('fs');
+const { resolve } = require('path');
+const { graphql } = require('@octokit/graphql');
+const { Promise } = require('node-fetch');
+const { Queries } = require('./issue-queries');
+require('dotenv').config({
+  path: resolve(process.cwd(), '.env.local'),
+});
+
```

Comment:
consider renaming the file to `build-dashboard` so it refers to what is built, and not what data is displayed there